### PR TITLE
replace qtchooser with lrelease. fixes issue #293

### PR DESCRIPTION
**Pull request by realchrisolin**
This should be self-explanatory. In lieu of a more elegant way of fixing this, I opted for the "sledgehammer approach".

### CODE REVIEW

**Comment by danieleds** [comment]
Thanks.

`qtchooser` is used as a default command in `./configure`... Don't you get an error when you run `./configure` too?

**Comment by realchrisolin** [comment]
strangely, no. I'll try reasoning out why and comment back.

**Comment by realchrisolin** [comment]
It's because I used the notepadqq-git AUR package, which doesn't run ./configure:

build() {
        cd "${_pkgname}"

         qmake-qt5 PREFIX=/usr notepadqq.pro

         make

}

Would you prefer I fix it accordingly in this PR before merging?

**Comment by danieleds** [comment]
> Would you prefer I fix it accordingly in this PR before merging?

Yeah, thanks!

**Comment by realchrisolin** [comment]
Done. I've only tested this on Arch (successfully)and can't guarantee these changes won't break something else on other distributions. However, they shouldn't as long as qmake-qt5 and lrelease are available.

**Comment by realchrisolin** [comment]
Also, the Travis CI build appeared to have failed because the `./configure` command didn't include the `--prefix /usr` argument, which is noted in the readme.

File: configure [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 # Don't set `-e'!
 
-test -n "$QMAKE"    || QMAKE="qtchooser -run-tool=qmake -qt=5"
-test -n "$LRELEASE" || LRELEASE="qtchooser -run-tool=lrelease -qt=5"
+test -n "$QMAKE"    || QMAKE="qmake-qt5"
```
Comment (danieleds) [inline configure]: Put just the standard `qmake` here... then, if an alias is needed, users can specify it by manually setting the environment variable.

Comment (realchrisolin) [inline configure]: Done, sorry for the delay.

**Comment by danieleds** [comment]
Sorry for the delay.
Travis fails because, on Travis, "qmake" is using Qt 4 and without lrelease we don't have a way to reliably use Qt 5 (qmake-qt5 is not always available with that name).
For now, I'd say that the best workaround is the one specified [here](https://github.com/notepadqq/notepadqq/issues/293#issuecomment-252586049) unless there is some problem with it. If that is the case, let me know!

**Comment by hasufell** [comment]
The suggested workaround doesn't work for me. Even when setting QMAKE and LRELEASE qtchooser is invoked.

**Comment by realchrisolin** [comment]
Out of curiosity (and to help others that might run into the same issue), what distribution and release version are you using?

**Comment by hasufell** [comment]
I've packaged it for the souce distro exherbo: 
https://github.com/hasufell/hasufell-repository/blob/master/packages/app-editors/notepadqq/notepadqq-1.0.1.exheres-0
http://git.exherbo.org/summer/packages/app-editors/notepadqq/index.html

**Pull request closed without merging**